_posts/2022-11-07-changelog-182.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #182"
+date: 2022-11-07 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9650] Increase the minimal supported version of Rust to 1.56
+
+* Procedural macro support improvements:
```

Comment:
Let's add a usual note thta this feature is not enabled by default yet and what users should do to enable it